controllers/productController.js

```diff
@@ -83,8 +85,59 @@ const getProduct = async (req, res) => {
   }
 };
 
+const deleteP = async (req, res) => {
+  const del_id = req.params.id;
+
+  try {
+    await deleteProduct(id);
+
+    logger.info('Product deleted successfully');
+
+    res.status = SUCCESS_NO_CONTENT;
+    res.send(SUCCESS_MESSAGE);
+    res.status = SUCCESS;
+  } catch (error) {
+    const message = error.message || INTERNAL_ERROR_MESSAGE;
+    const status = error.statusCode || INTERNAL_SERVER;
+
+    logger.error(message);
+
+    res.send(makeError(message, status));
+  }
+};
+
+const updateP = async (req, res) => {
+  const up_id = req.params.id;
+
+  data = {
+    name: req.body.name,
+    description: req.body.description,
+    price: req.body.price,
+    category: req.body.category,
+    size: req.body.size,
+    image: req.body.image
+  };
+
```

Comment:
Since req.body = data we can simnply write it as : const data=req.body

Reply:
done